Realm/RealmSwift/Aliases.swift

```diff
@@ -20,5 +20,11 @@ import Realm
 
 // These types don't change when wrapping in Swift
 // so we just typealias them to remove the 'RLM' prefix
+
+// MARK: Aliases
+
+/// An enum of the different property types supported by Realm.
 public typealias PropertyType = RLMPropertyType
+
+/// A token that holds onto the Realm and the notification block.
```

Comment:
"An opaque token used to unregister notification blocks from Realms."

The fact that it has a reference to the RLMRealm and the block isn't exposed in the interface.